sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -24,7 +24,7 @@
 from argparse import ArgumentParser
 import json
 from multiprocessing import Pool
-from os import environ
+from os import environ, path
```

Comment:
I suggest to use pathlib.Path instead for consistency with the rest of the script 
```suggestion
from os import environ
```